.vscode/tasks.json

```diff
@@ -0,0 +1,32 @@
+{
+	"version": "2.0.0",
+	"tasks": [{
+		"label": "Start dev container",
+		"type": "shell",
+		"command": "docker compose --file .devcontainer/docker-compose.yaml up",
+		"windows": {
+		  "command": "docker compose --file .devcontainer\\docker-compose.yaml up"
+		},
+		"group": "none",
+		"presentation": {
+		  "reveal": "always",
+		  "panel": "new"
+		},
+		"runOptions": {
+		  "runOn": "folderOpen"
+		}
+	  }, {
+		"label": "Stop dev container",
+		"type": "shell",
+		"command": "docker compose --file .devcontainer/docker-compose.yaml down",
+		"windows": {
+		  "command": "docker compose --file .devcontainer\\docker-compose.yaml down"
+		},
+		"group": "none",
+		"presentation": {
+		  "reveal": "always",
+		  "panel": "new"
+		}
```

Comment:
Do you know if the "start" task is automatically stopped (sigint'd?) when VS code is closed? I think it's best if we ensure that the containers do not continue running in the background. If that is not the case, I'm not sure what we should do since there's nothing equivalent to e.g. `folderClose` (see https://github.com/microsoft/vscode/issues/65986). Let me know @Thechi2000, thanks! 👍

Reply:
This does not seem to be possible, but it seems that reopening the folder does not launch more containers; the existing one are reattached into the task. I think we should leave it that way, since there is not much we could do